reconcile/test/utils/test_early_exit_cache.py

```diff
@@ -0,0 +1,185 @@
+import datetime
+from typing import Any
+from unittest.mock import create_autospec
+
+import pytest
+from deepdiff import DeepHash
+from pytest_mock import MockerFixture
+
+from reconcile.utils.early_exit_cache import (
+    CacheKey,
+    CacheStatus,
+    CacheValue,
+    EarlyExitCache,
+)
+from reconcile.utils.secret_reader import SecretReaderBase
+from reconcile.utils.state import State
+
+
+@pytest.fixture
+def state() -> Any:
+    return create_autospec(State)
+
+
+@pytest.fixture
+def secret_reader() -> Any:
```

Comment:
Do you mind and use the existing [secret_reader fixture](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/conftest.py#L40) instead?

Reply:
Nice! Reused in https://github.com/app-sre/qontract-reconcile/pull/3995/commits/ef28c2980b7c8b83726791998f4c25370ad3dcec